src/braket/circuits/observables.py

```diff
@@ -311,11 +319,26 @@ def __init__(self, matrix: np.ndarray, display_name: str = "Hermitian"):
 
         super().__init__(qubit_count=qubit_count, ascii_symbols=[display_name] * qubit_count)
 
-    def to_ir(self) -> List[List[List[List[float]]]]:
+    def to_jaqcd(self) -> List[List[List[List[float]]]]:
         return [
             [[[element.real, element.imag] for element in row] for row in self._matrix.tolist()]
         ]
 
+    def to_openqasm(self, qubit_reference_format: str, target: QubitSet = None) -> str:
+        def _serialized_matrix():
```

Comment:
I generally find functions-in-functions to be confusing. Why not move it out?

Reply:
Since this is only applicable for this function, and not the class or any other method in the class, I scoped it to the function. Happy to move it out if you feel strongly about it.

Reply:
I think if this becomes the norm it will make our code much more messy & hard to understand. As a code style thing, I think you would be better suited to know.

Reply:
Updated